source/cloud/cloud-administration/sso-google.md

```diff
@@ -4,41 +4,41 @@ Follow these steps to configure Mattermost to use Google as a Single Sign-on (SS
+Once in the API Library, scroll down to [Google People API](https://console.developers.google.com/apis/api/plus/overview) and select **Enable** in the header. This might take a few minutes to propagate through Google's systems.
 
-3. Leave the Google+ API menu to return to the **APIs & Services menu**. In the left hand sidebar select **Credentials**, click the **OAuth consent screen** header and enter "Mattermost" as the **Application Name**, then click **Save**.
+3. Leave the Google+ API menu to return to the **APIs & Services menu**. In the left hand sidebar select **Credentials**, select the **OAuth consent screen** header and enter "Mattermost" as the **Application Name**, then select **Save**.
 
-![](../../../source/images/google_3_oauth_consent_screen.png)
+  ![](../../../source/images/google_3_oauth_consent_screen.png)
 
 Scroll further down to add your Mattermost instance to a list of authorized domains that can access the API. For example, if it is hosted on `mattermost.yourdomain.com` add `yourdomain.com`.
 
-![](../../../source/images/google_authorised_domains.png)
+  ![](../../../source/images/google_authorised_domains.png)
 
-4. Select **Credentials** and click **Create credentials**, then choose **OAuth client ID** from the drop-down list.
+4. Select **Credentials** and select **Create credentials**, then choose **OAuth client ID** from the drop-down list.
 
-![](../../../source/images/google_3_oauth_client_id.png)
+  ![](../../../source/images/google_3_oauth_client_id.png)
 
-5. Select **Web Application** as the **Application type**, and choose a descriptive **Name** for the OAuth connection.
+5. Select **Web Application** as the **Application type**, and enter a descriptive **Name** for the OAuth connection.
 
-Under **Restrictions > Authorized redirect URIs**, enter `your-mattermost-url/signup/google/complete` (example: `http://localhost:8065/signup/google/complete`). Click **Create**.
+Under **Restrictions > Authorized redirect URIs**, enter `your-mattermost-url/signup/google/complete` (example: `http://localhost:8065/signup/google/complete`). select **Create**.
 
-![](../../../source/images/google_4_web_app.png)
+  ![](../../../source/images/google_4_web_app.png)
 
 6. Copy the **Client ID** and **Client Secret**, which will be used to configure Google SSO for Mattermost.
 
-![](../../../source/images/google_5_client_id_secret.PNG)
+  ![](../../../source/images/google_5_client_id_secret.PNG)
 
-7. In **System Console > OAuth 2.0 > Select OAuth 2.0 service provider**, choose **Google Apps** as the service provider and enter **Client ID** and **Client Secret** from step 6 in their respective fields.
+7. In **System Console > OAuth 2.0 > Select OAuth 2.0 service provider**, choose **Google Apps** as the service provider, then enter **Client ID** and **Client Secret** from step 6 in their respective fields.
 
 8. Restart your Mattermost server to see the changes take effect.
```

Comment:
@marianunez - Given that Cloud admins can't restart a workspace, can admins set up OAuth 2.0 as documented without a restart?

Reply:
Not sure, @jwilander do we support OAuth 2.0 and how do their changes take effect after setup?

Reply:
We do support it and it shouldn't require a restart anymore, I think that's outdated and from a legacy version. I recently set up Google SSO with OAuth 2.0 and it worked fine, no restart needed.

Reply:
Thanks! I've removed that step since it's no longer necessary.

Reply:
Do we still need to remove this step for refresh? It still seems to be showing for me

Reply:
@marianunez Due to a merge conflict, the server restart step was still visible. It _should_ be removed now.

Reply:
Not sure if I'm looking at the wrong thing but I still see in this PR files diff that this line has not been modified/deleted

`8. Restart your Mattermost server to see the changes take effect.`